websites/C/Codesandbox.io/presence.ts

```diff
@@ -0,0 +1,96 @@
+const presence = new Presence({
+		clientId: "961652082027421746"
+	}),
+	browsingTimestamp = Math.floor(Date.now()),
+	formats = [
+		"js",
+		"jsx",
+		"ts",
+		"tsx",
+		"json",
+		"py",
+		"cpp",
+		"cs",
+		"c",
+		"py",
+		"swift",
+		"java",
+		"html",
+		"css",
+		"php",
+		"kt",
+		"sql",
+		"lua",
+		"ru",
+		"bash",
+		"sh",
+		"bat",
+		"cr",
+		"go"
+	];
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "load",
+			smallImageKey: "smallimagekey",
+			smallImageText: "Codesandbox",
+			details: "Loading",
+			startTimestamp: browsingTimestamp
+		},
+		{ pathname, search } = window.location;
+	if (pathname.includes("/dashboard/")) {
+		presenceData.details = "Looking at their dashboard";
+		presenceData.largeImageKey = "dashboard";
+	} else if (pathname.includes("/u/")) {
+		const uname = document.querySelector<HTMLSpanElement>(
+			'[class="sc-bdnylx sc-gtssRu gDXMLZ itZLEx"]'
+		);
+		presenceData.details = "Looking at a user's profile";
+		presenceData.state = `@${uname.textContent}`;
+		presenceData.buttons = [
+			{
+				label: `View @${uname.textContent}'s Profile`,
```

Comment:
This wouldn't be recommended as: 
1. You already show the user's name in L48 (presenceData.state)
2. This could possibly hit the rate limit due 

I would recommend the following instead
```suggestion
				label: "View Profile",
```